content/demos/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Demos"
+description: ""
+featured_image: '/images/opi_logos/backgrounds/OPI_logo_social_dark.png'
+menu:
+  main:
+    weight: 1
+---
+
+## Networking Cloud Demo
+
+* [Video recording](https://youtu.be/UWIGLShe8d0)
+* [API protocol buffers](https://github.com/opiproject/opi-api/tree/main/network/cloud)
+* clients <https://github.com/opiproject/pydpu> and <https://github.com/opiproject/godpu>
```

Comment:
nit, I'd capitalize the first letter of each bullet.

Reply:
will do